src/vue/pages/Limits.vue

```diff
@@ -0,0 +1,270 @@
+        <!-- hack with key allows us to reload data -->
+        <collection-loader
+          :key="`request-reloader-${currentRequestsReloadingCount}`"
+          :first-page-loader="loadLimitsRequests"
+          :page-limit="limitsRequestsQueries.limit"
+          @first-page-load="setLimitsRequests"
+          @next-page-load="extendLimitsRequests"
+        />
+      </div>
+    </div>
+
+    <drawer :is-shown.sync="isLimitsChangeDrawerShown">
+      <template slot="heading">
+        {{ 'limits.limits-form-heading' | globalize }}
+      </template>
+      <limits-form
+        @limits-changed="limitsChanged"
+        :limits="selectedLimitsList"
+      />
+    </drawer>
+  </div>
+  <div v-else>
+    <no-data-message
+      :title-id="'limits.no-limits-list'"
+      :message-id="'limits.here-will-limits-list'"
+    />
+  </div>
+</template>
+
+<script>
+import SelectField from '../fields/SelectField'
+import Drawer from '@/vue/common/Drawer'
+import TopBar from '@/vue/common/TopBar'
+import LimitsForm from '@/vue/forms/LimitsForm'
+import LimitsTableRenderer from '@/vue/pages/Limits/Limits.TableRenderer'
+import LimitsRequestsListRenderer from '@/vue/pages/Limits/Limits.RequestsListRenderer'
+import NoDataMessage from '@/vue/common/NoDataMessage'
+import { Sdk } from '@/sdk'
+import { mapGetters, mapActions } from 'vuex'
+import { vuexTypes } from '@/vuex'
+import { vueRoutes } from '@/vue-router/routes'
+import { ACCOUNT_TYPES, STATS_OPERATION_TYPES } from '@tokend/js-sdk'
+import { ErrorHandler } from '@/js/helpers/error-handler'
+import { LimitsRecord } from '@/js/records/entities/limits.record'
+import { LimitsUpdateRequestRecord } from '@/js/records/requests/limits-update.record'
+import config from '@/config'
+import CollectionLoader from '@/vue/common/CollectionLoader'
+
+export default {
+  name: 'limits',
+  components: {
+    SelectField,
+    Drawer,
+    TopBar,
+    LimitsForm,
+    LimitsTableRenderer,
+    LimitsRequestsListRenderer,
+    CollectionLoader,
+    NoDataMessage,
+  },
+  data: () => ({
+    selectedAsset: '',
+    isLimitsChangeDrawerShown: false,
+    vueRoutes,
+    ACCOUNT_TYPES,
+    formattedAccountLimits: {},
+    isLimitsLoading: false,
+    isLimitsRequestsLoading: false,
+    limitsRequests: [],
+    limitsRequestsQueries: {
+      limit: config.REQUESTS_PER_PAGE,
+      order: 'desc',
+    },
+    currentRequestsReloadingCount: 0,
+  }),
+  computed: {
+    ...mapGetters([
+      vuexTypes.accountBalances,
+      vuexTypes.accountTypeI,
+      vuexTypes.accountId,
+    ]),
+    accountBalancesAssetsCodes () {
+      return this.accountBalances.map(i => i.asset)
+    },
+    selectedLimitsList () {
+      return this.formattedAccountLimits[this.selectedAsset] || {}
+    },
+  },
+  async created () {
+    if (!this.accountBalances.length) await this.loadCurrentBalances()
+    if (this.accountBalancesAssetsCodes.length) this.setDefaultAssetCode()
+    this.loadLimits()
```

Comment:
aggressive error shown in case of failure

Reply:
What does it mean?